tests/rados/mute_alerts.py

```diff
@@ -0,0 +1,289 @@
+import logging
+import re
+import time
+import traceback
+
+from ceph.ceph_admin import CephAdmin
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Polarion ID: https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573854
+    Feature : Mute Health alerts : https://trello.com/c/AU8FT6Qp/27-mute-health-warnings
+    RFE Bug : https://bugzilla.redhat.com/show_bug.cgi?id=1821508
+    upstream doc : https://docs.ceph.com/en/octopus/rados/operations/monitoring/#muting-health-checks
+
+    1. Check the cluster health status and simulate various failures.
+    2. Verify the health warning/ error generated due to the failures.
+    3. Mute the failures, test the various arguments and verify that the alerts are muted successfully
+    Args:
+        ceph_cluster (ceph.ceph.Ceph): ceph cluster
+    """
+    log.info("Running test for feature : Mute Health Alerts")
+    log.info(run.__doc__)
+    config = kw.get("config")
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    all_alerts = get_alerts(cephadm)
+    alert_list = ["MON_DISK_BIG", "OSDMAP_FLAGS"]
+    if all_alerts["active_alerts"]:
+        log.info(
+            f"There are health alerts generated on the cluster. Alerts : {all_alerts['active_alerts']}"
+        )
+    else:
+        log.info("Cluster Health is ok. \n Generating a alert to verify feature")
+
+    # Scenario 1 : Verify the auto-unmute of alert after TTL
```

Comment:
I like the comments added to each section.

IMO, we should have separate functions executing the `scenario`. What do you think?

Reply:
Added methods that can execute scenario on the alerts provided.